src/other-reprs.md

```diff
@@ -112,26 +112,12 @@ assert_eq!(16, size_of::<MyReprOption<&u16>>());
 
 This optimization still applies to fieldless enums with an explicit `repr(u*)`, `repr(i*)`, or `repr(C)`.
 
-## repr(packed)
+## repr(packed(n))
 
-`repr(packed)` forces Rust to strip any padding, and only align the type to a
-byte. This may improve the memory footprint, but will likely have other negative
-side-effects.
+`repr(packed(n))` forces Rust to strip any padding, and align the type to a [`n`-byte boundary][rust-reference-type-layout],
```

Comment:
I don't believe this is quite correct.  The original of `#[repr(packed)]` is equivalent to `#[repr(packed(1))]` which does remove padding.  However, a value greater than 1 can retain some padding, so i think it isn't quite correct to say it strips *all* padding.